output.js

```diff
@@ -276,19 +325,38 @@ function formatError(config, err) {
         diff += generateDiff(config, err);
     }
 
+    /**
+     * The nearest location where the user's code triggered the error.
+     * @type {import('errorstacks').StackFrame}
+     */
+    let nearestFrame;
+
     const stack = errorstacks.parseStackTrace(err.stack)
         .map(frame => {
             if (frame.name) {
+                // Only show frame for errors in the user's code
+                if (!nearestFrame && !/node_modules/.test(frame.fileName)) {
+                    nearestFrame = frame;
+                }
+
                 const location = link(config, frame.fileName, `file://${frame.fileName}`);
-                return color(config, 'dim', `    at ${frame.name} (`) + color(config, 'cyan', location) + color(config, 'dim', `:${frame.line}:${frame.column})`);
+                return color(config, 'dim', `  at ${frame.name} (`) + color(config, 'cyan', location) + color(config, 'dim', `:${frame.line}:${frame.column})`);
             } else {
                 // Internal native code in node
                 return color(config, 'dim', frame.raw);
             }
         });
 
+    
+    let codeFrame = '';
+    if (nearestFrame) {
+        const { fileName, line, column } = nearestFrame;
+        const content = fs.readFileSync(fileName, 'utf-8');
```

Comment:
Wouldn't this block the whole execution? I'd rather make the function `async` and use `await` here. Sure, it's not a (big) problem on most systems, but when running tests on an NFS system, reading the file can take quite some time.

Reply:
Good idea, I was unsure initially, but I didn't think of network shares. Thanks for the idea! :+1: